api/Node.json

```
@@ -738,10 +738,10 @@
               "version_added": true
             },
             "safari": {
-              "version_added": true
+              "version_added": "≤4"
```

```suggestion
              "version_added": "1"
```

Based on https://trac.webkit.org/browser/webkit/trunk/WebCore/khtml/ecma/kjs_dom.lut.h?rev=4552